go/beacon/api/api.go

```diff
@@ -35,6 +36,15 @@ var (
 	// ErrBeaconNotAvailable is the error returned when a beacon is not
 	// available for the requested height for any reason.
 	ErrBeaconNotAvailable = errors.New(ModuleName, 2, "beacon: random beacon not available")
+
+	// MethodSetEpoch is the method name for setting epochs.
+	MethodSetEpoch = transaction.NewMethodName(ModuleName, "SetEpoch", EpochTime(0))
```

Comment:
This change renames module name from `AppName = 000_beacon` to `ModuleName = beacon`. I think that in production this should not be a breaking change, even though we now return a different error when `SetEpoch` transaction is executed.

Reply:
`SetEpoch` is a debug-only option which cannot be enabled in production, so it should be fine if there are no other side effects.

Reply:
Oh I see, I think error messages are excluded as long as module and error code are the same. And in this case the module would be unknown and error code the default of 1.